x-pack/plugins/observability_solution/apm/public/services/telemetry/telemetry_events.ts

```diff
@@ -33,4 +33,54 @@ const searchQuerySubmittedEventType: TelemetryEvent = {
   },
 };
 
-export const apmTelemetryEventBasedTypes = [searchQuerySubmittedEventType];
+const entityExperienceStatusEventType: TelemetryEvent = {
+  eventType: TelemetryEventTypes.ENTITY_EXPERIENCE_STATUS,
+  schema: {
+    status: {
+      type: 'keyword',
+      _meta: {
+        description: 'The status of the Entity experience (Enabled or Disabled)',
+      },
+    },
+  },
+};
+
+const entityInventoryPageStateEventType: TelemetryEvent = {
+  eventType: TelemetryEventTypes.ENTITY_INVENTORY_PAGE_STATE,
+  schema: {
+    state: {
+      type: 'keyword',
+      _meta: {
+        description: 'The current entity inventory page state (empty_state or available)',
+      },
+    },
+  },
+};
+
+const entityInventoryAddDataEventType: TelemetryEvent = {
+  eventType: TelemetryEventTypes.ENTITY_INVENTORY_ADD_DATA,
+  schema: {
+    type: {
+      type: 'keyword',
```

Comment:
The `type` is a bit confusing as property name. I was expecting it sends the type of the event when I was reading the code. 

It will send
```
properties: {
   type: empty_state,
   journey: add_apm_agent
}
```
is it just me ? 🤔

Reply:
WDYT if I change it to `view`?
```
properties: {
   view: empty_state/add_data_button/add_apm_cta,
   journey: add_apm_agent
}

```

Reply:
much better. Thank you